RELEASING.md

```diff
@@ -0,0 +1,8 @@
+# Sceptre release process
+
+1. Bump version in __init__.py & setup.cfg
```

Comment:
I think you should give a full filepath for `__init__.py`, since it's not immediately clear which one that refers to.

Reply:
Done.